docs/reference/contributing/tools/mbedls.md

```diff
@@ -1,21 +1,21 @@
-### mbed-ls
+### Mbed-ls
 
-[Add description here.]
+Mbed-ls is a Python module that detects and lists Mbed Enabled boards connected to a host computer.
 
-#### Assumptions
+### Defined behavior
 
-##### Defined behavior
+Mbed-ls needs the following information to correctly detect an Mbed Enabled target:
 
-[Add defined behavior. A bulleted list is fine.]
+ - A four-digit hexadecimal identifier of a board class.
+ - A vendor string for detection on Windows.
+ 
+Further, Mbed-ls only detects devices that meet the following criteria:
 
-##### Undefined behavior
+ - Devices have a mass storage device (MSD) endpoint.
+ - The MSD endpoint has a FAT file system.
+ 
+### Implementation
 
-[Add undefined behavior. A bulleted list is fine.]
+When adding a target to Mbed-ls, you typically need to add a new definition to the platform database. You can add a new target to the platform database by adding entry mapping from your platform's four-digit hexadecimal identifier to the name of the board to the `DEFAULT_PLATFORM_DB` variable in [`mbed_lstools/platform_database.py`](https://github.com/ARMmbed/mbed-ls/blob/master/mbed_lstools/platform_database.py).
```

Comment:
platform replaced with board. Where do you get a board id from?

Reply:
"platform database" is the thing in mbed-ls. I will change "your platform's four-digit..." to "your board's four-digit..." and add where to find the "four-digit hexadecimal identifier". It's the "Unique product code" on the os.mbed.com website which you should receive through mbed enabled.